src/app/atoms/null-component.tsx

```diff
@@ -0,0 +1 @@
+export const NullComponent = () => null;
```

Comment:
pls move this to `lib/ui` + reuse in some more places, search for "const EmptyHeader = () => null" in the project